events/models.py

```diff
@@ -29,6 +29,7 @@ class Event(models.Model):
         default=True,
         help_text="If this is checked all users that attends the event needs to \
         be accepted by an admin.")
+    external_signup_url = models.CharField(max_length=80, blank=True)
```

Comment:
Yeah, that's what I've found to. It sets the validators.URLValidators as it's default validator (as can be seen here https://github.com/django/django/blob/2f9861d823620da7ecb291a8f005f53da12b1e89/django/db/models/fields/__init__.py#L2282 )

Anyway, I think we'll be fine with using the URLField :)